app/src/main/java/ar/scacchipa/twittercloneapp/presentation/login/FragmentLoginWebSection.kt

```diff
@@ -95,4 +96,31 @@ class FragmentLoginWebSection : Fragment() {
         super.onDestroyView()
         binding = null
     }
+
+    private fun getConsumableAuthCode(): String {
```

Comment:
I think it is ok not to have this in a usecase, but I would put that logic in the viewmodel, not in the ui.

Reply:
prueba DIana

Reply:
It's ready

Reply:
Trying to write a comment without sending